### DISCUSSION TITLE
osu!lazer multiplayer is not loading! Is it done on purpose?

### DISCUSSION THREAD

**Discussion by hazelcode** [ppy/osu#19712] [Q&A] [answered]
![osu_2022-08-11_11-56-02](https://user-images.githubusercontent.com/84425557/184190815-d832313d-7d11-48ad-a18c-2f81002e1a9f.png)

I need help to test multiplayer.

Is this an issue? Or is it done on purpose?
How can I unlock the multiplayer or when will be unlocked?

The version that I am using is the latest, 2022.810.2

**Comment by TheFantasticLoki** [comment]
is there a chance you can submit some speedtest.net results and have you restarted your computer? Doing a full restart of your computer often fixes many weird issues like this but I just tested latest version of lazer and I have no problems loading the multiplayer screen nor it's lobbies so please double check your connection and restart your computer and see if the problems persist.

**Reply by hazelcode** [reply]
<a href="https://www.speedtest.net/result/13537097850"><img src="https://www.speedtest.net/result/13537097850.png"/></a>

**Reply by hazelcode** [reply]
Is this good? (latin america conection, sorry)

**Reply by TheFantasticLoki** [reply]
I think it "Should" be more than enough for lazer multiplayer but overall that is not a good connection. But before I explain you should also listen to peppy and provide all the .log files that are inside the osu logs folder. In the game go to settings and there should be an option to open the osu folder. But for your connection the difference in ping times is very concerning as it seems to be constantly increasing according to your speed test. The ping results should be within like 5ms of each other but yours are basically doubling themselves each test which could cause issues. Please try with a wired connection if available and also provide logs.

**Reply by hazelcode** [reply]
Yes, i am using an ethernet connection

**Comment by peppy** [comment, marked as answer]
Please [provide logs](https://github.com/ppy/osu/discussions/15188) to help us understand your issue.

**Reply by hazelcode** [reply]
[database.log](https://github.com/ppy/osu/files/9332577/database.log)
[network.log](https://github.com/ppy/osu/files/9332578/network.log)
[performance.log](https://github.com/ppy/osu/files/9332579/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9332580/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9332581/updater.log)

**Reply by peppy** [reply]
Based on logs you're having SSL issues.

Hard to say what the cause is, but something is up with your PC configuration. If you've done anything dodgy with your network settings, reset them. If you don't know how to fix it, I'd recommend formatting your PC as you may be compromised.

**Reply by hazelcode** [reply]
Yes. I tested the multiplayer from my cellphone with the same connection and worked very fine. The problem is my PC. I will do my best to resolve that problem. Thanks, Peppy.
